packages/gasket-plugin-mocha/generator/test/register-loader.js

```diff
@@ -0,0 +1,4 @@
+import { register } from 'module';
+import { pathToFileURL } from 'url';
+
+register('./test/node-loader-babel.js', pathToFileURL('./test'));
```

Comment:
This is great!

I may have misunderstood your question the other day about forking or copying. However, seeing this now, instead of having a loose `node-loader-babel.js` generated for each app, could this be imported from a common package, i.e.
```suggestion
register('@gasket/plugin-mocha/loader', pathToFileURL('./test'));
```

or some such name?